drivers/hmis/app/graphql/mutations/update_form_definition.rb

```diff
@@ -16,6 +16,7 @@ def resolve(id:, input:)
 
       definition = Hmis::Form::Definition.find_by(id: id)
       raise 'not found' unless definition
+      raise 'only allowed to modify draft forms' unless definition.status == Hmis::Form::Definition::DRAFT
```

Comment:
might be nice to have a helper like `definition.draft?` since we check this in both mutations